quality/postman/emodb_tests_uac_role_update_role.postman_collection.json

```diff
@@ -99,7 +99,7 @@
 																"header": [],
 																"body": {
 																	"mode": "raw",
-																	"raw": ""
+																	"raw": "{\n    \"permissions\": [\n        \"et eu amet eiusmod\",\n        \"velit an\"\n    ],\n    \"description\": \"ex ut cupidatat quis\",\n    \"name\": \"quis elit amet\"\n}"
```

Comment:
here and below
are these permissions valid?

Reply:
This is coming from swagger file. Anyway, cleaned up.